### ISSUE TITLE
Y axis min & max should consider template variables defined values

### ISSUE THREAD

**Issue by doddahulugappa** [grafana/grafana#97998]
**Why is this needed**:

This is required because when we want to see data beyond the scale limits we set

**What would you like to be added**:

By Allowing Y Min & Y Max field to receive template variables, it can be achieved.

**Who is this feature for?**
This is required for all the users who want to have dynamic y scale.

**Comment by xkrogen** [comment]
Along this same vane, would love to see other axis configurations being able to be pulled from variables, such as the scale. I have some use cases where depending on which filters are applied, you may want to view the data as linear scale or log scale (different inputs have very different ranges requiring the different scale), and being able to swap the scale alongside the input variables would be hugely useful.

**Comment by leeoniya** [comment]
yes, we have a plan to support variables in all panel options and field config / overrides. we did this recently for transformations. probably in the next few months.
